incubator/spark/templates/spark-worker-deployment.yaml

```diff
@@ -34,3 +34,8 @@ spec:
             requests:
               cpu: "{{.Values.Worker.Cpu}}"
               memory: "{{.Values.Worker.Memory}}"
+          env:
+          - name: SPARK_DAEMON_MEMORY
+            value: {{ default "1g" .Values.Worker.DaemonMemory | quote }}
+          - name: SPARK_WORKER_MEMORY
+            value: {{ default "1g" .Values.Worker.WorkerMemory | quote }}
```

Comment:
Should this be `.Values.Worker.Memory` or `.Values.Worker.ExecutorMemory`? I can't see `.Values.Worker.WorkerMemory` in values.yaml.